starport-lambda/src/main/scala/com/krux/starport/lambda/SubmitHandler.scala

```diff
@@ -33,41 +32,59 @@ class SubmitHandler extends RequestHandler[SubmitRequest, SubmitResponse] with L
     var outString = ""
     var errString = ""
 
-    try {
-      SubmitPipeline.main(input.getArgs)
-    } catch {
-      case caughtExit: LambdaExitException => {
-        status = caughtExit.status
-        logger.error("exit:", caughtExit)
-        logger.error(scanTmpFiles())
-      }
-      case unhandled: Throwable =>  {
-        status = 255
-        logger.error("exception:", unhandled)
-        logger.error(scanTmpFiles())
-      }
-    } finally {
-      outPrintStream.flush()
-      errPrintStream.flush()
-      outString = outCapture.toString
-      errString = errCapture.toString
-      outCapture.reset()
-      errCapture.reset()
-      lambdaOut.print(outString)
-      lambdaErr.print(errString)
-    }
+    val args: Array[String] = input.getArgs
+
+        try {
+          args.head match {
+            case "deleteTmpDir" => {
+              status = 255
+              logger.error("received call to delete /tmp/.starport")
+              logger.error(deleteTmpDir())
+            }
+            case _ => SubmitPipeline.main(args)
+          }
+        } catch {
+          case caughtExit: LambdaExitException => {
+            status = caughtExit.status
+            logger.error("exit:", caughtExit)
+            logger.error(scanTmpFiles())
+          }
+          case unhandled: Throwable => {
+            status = 255
+            logger.error("exception:", unhandled)
+            logger.error(scanTmpFiles())
+          }
+          } finally {
+            outPrintStream.flush()
+            errPrintStream.flush()
+            outString = outCapture.toString
+            errString = errCapture.toString
+            outCapture.reset()
+            errCapture.reset()
+            lambdaOut.print(outString)
+            lambdaErr.print(errString)
+          }
+
+          SubmitResponse(outString, errString, status, input)
 
-    SubmitResponse(outString, errString, status, input)
   }
 
   /**
    * @return /tmp file listing with size for troubleshooting purposes
    */
   private def scanTmpFiles(): String = {
-    new File("/tmp")
+    new File("/tmp/.starport")
```

Comment:
I mean it's referenced here too. We already accept different tmp directories. Define a constant val, and reference it in both places

Reply:
Ah gotcha. I'll fix that now.